### PR TITLE
Create Blogto.com.xml

### PR DESCRIPTION
**Pull request by csavard-sudo**

### CODE REVIEW

**Review by zoracon — CHANGES_REQUESTED** [review]
Looks good! Could use a few other subdomains.

Please add 
`media.blogto.com`
`contests.blogto.com`

**Comment by Nintendofan885** [comment]
> Looks good! Could use a few other subdomains.
> 
> Please add
> `media.blogto.com`
> `contests.blogto.com`

contests.blogto.com seems to give a 'host not resolved error'

**Comment by zoracon** [comment]
> > Looks good! Could use a few other subdomains.
> > Please add
> > `media.blogto.com`
> > `contests.blogto.com`
> 
> contests.blogto.com seems to give a 'host not resolved' error

I am seeing that as well, you can remove that

File: src/chrome/content/rules/Blogto.com.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<ruleset name="Blogto.com">
+	<target host="blogto.com" />
+	<target host="www.blogto.com" />
+	<target host="media.blogto.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Blogto.com.xml:4]: I also found `api.blogto.com` and `patios.blogto.com`

**Review by csavard-sudo — COMMENTED** [review]
done :)

**Review by J0WI — APPROVED** [review]

**Pull request merged**